lua/neotest-rust/init.lua

```diff
@@ -104,7 +104,7 @@ local function integration_test_name(path)
 end
 
 function adapter.discover_positions(path)
-    local query = [[
+    local query = [[;; query
```

Comment:
what does this change do?

Reply:
That's just for treesitter injection to highlight it as `query`:

https://github.com/nvim-treesitter/nvim-treesitter/blob/ec97c5ec720da7b8bb5f53572c91c5b144ae0cae/queries/lua/injections.scm#L26-L27

I can remove that if you want.

Reply:
oh that's cool, you can leave it in